documentation/docs/guide/SiteEditor/Patterns/DSWPCardWithHyperLinkList.md

```diff
@@ -0,0 +1,84 @@
+# DSWP Card with Hyperlink List
+
+This card-based layout pattern is ideal for presenting multiple related items, such as services, features, or categories, in an organized grid format. It provides users with clear navigation options through linked lists while maintaining visual consistency across your WordPress site.
+
+> **Important**: Each pattern consists of several blocks. It's crucial to only adjust the content and color block settings, as the structural settings like padding and margins are designed to ensure the pattern remains responsive. Altering the structural block settings may cause the pattern to function improperly.
+
+Each Design System Pattern enables you to customize the content and styles of individual blocks. For instance, here's how you can change the separator color on a card.
+
+### **Step 1**: Click the plus button.
+![Step 1](/images/patterns/dswp-card-with-hyperlink-list/a.png)  
+
+### **Step 2**: Click "All".
+![Step 2](/images/patterns/dswp-card-with-hyperlink-list/b.png)  
+
+### **Step 3**: Click the pattern "dswp-card-with-hyperlink-list".
+![Step 3](/images/patterns/dswp-card-with-hyperlink-list/c.png)  
+
+### **Step 4**: Click "Zoom" button to zoom into the page editor.
+![Step 4](/images/patterns/dswp-card-with-hyperlink-list/d.png)  
+
+### **Step 5**: Click "Block Settings" button.
+![Step 5](/images/patterns/dswp-card-with-hyperlink-list/e.png)  
+
+### **Step 6**: Click "Block overview (1)" then select the "Separator (2)" block.
+![Step 6](/images/patterns/dswp-card-with-hyperlink-list/f.png)  
+
+### **Step 7**: Click "Block Styles".
+![Step 7](/images/patterns/dswp-card-with-hyperlink-list/g.png)  
+
+### **Step 8**: Scroll down in the block styles and select the top border (1), then choose another color (2). 
+![Step 8](/images/patterns/dswp-card-with-hyperlink-list/h.png)  
+
+### **Step 9**: Don't forget to click save.
+![Step 9](/images/patterns/dswp-card-with-hyperlink-list/i.png)  
+
+## Pattern Layout Variations
+This pattern has different layout variations, each variation has been developed to accommodate 2, 3, and 4 cards. Consider the layout that best fits your content needs.
+![variations1](/images/patterns/dswp-card-with-hyperlink-list/k.png)
+
+## To remove a card from this pattern:
+
+1. Select the ellipses beside the card (column block) you would like to remove. 
+2. Select Delete. 
+
+![variations2](/images/patterns/dswp-card-with-hyperlink-list/l.png)
+
+## To add another card to this pattern:
+
+1. Select the ellipses beside any card (column block). (Which column you select is not important as you will be replacing its content.)
+2. Select Duplicate.
+
+## Before 
+![variations3](/images/patterns/dswp-card-with-hyperlink-list/m.png)  
+
+## After
+![variations4](/images/patterns/dswp-card-with-hyperlink-list/n.png)  
+
+## Pattern Blocks and Best Practices
```

Comment:
was all of the Pattern Blocks and Best Practices supposed to be in this documentation page?